Test.C

```diff
@@ -722,3 +730,19 @@ void sellingStore(struct Player *playerPtr, struct Equipment equipmentStock[NUM_
         scanf("%d", &equipSellSelect);
     }
 }
+
+/******************************************************************************
+* Function:    existsInArray
+* Description: Determines if an integer exists in an integer array
+* Parameters:  value, the value to search for
+*              arr, the array to search in
+* Return:      1 if it exists, 0 if it does not
+******************************************************************************/
+int existsInArray(int value, int arr[NUM_ROUNDS_WITH_ATTACK]) {
+    for(int i = 0; i < NUM_ROUNDS_WITH_ATTACK; i++) {
```

Comment:
No spaces in for per the style guide.
```suggestion
    for (int i=0; i<NUM_ROUNDS_WITH_ATTACK; i++) {
```